docs/tutorials/optimizers_conditionalgradient.ipynb

```diff
@@ -0,0 +1,558 @@
+{
```

Comment:
<p>Could you make this the function's docstring</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/590/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>